appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: '{build}'
+
+environment:
+    PATH:           'C:\Ruby200\bin;%PATH%'
+    VERSION_SUFFIX: -netstd

```

Comment:
Sorry, I didn't mean to ask for it, it just came along for the ride when I exported the YAML for the UI. I don't think we should ever set `VERSION_SUFFIX` in the YAML. It should be a clicky-clicky config item only.